edb/tools/gen_cast_table.py

```diff
@@ -161,14 +147,28 @@ def gen_cast_table():
     NAME - at the moment there's only one option 'edgeql'
     """
 
-    con = None
     try:
-        con = edgedb.connect(user=edgedb_defines.EDGEDB_SUPERUSER,
-                             database=edgedb_defines.EDGEDB_SUPERUSER_DB,
-                             port=5656)
-        main(con)
+        res = subprocess.run([
+            'edb',
+            'cli',
+            '--json',
```

Comment:
This would shortly become `edb cli query --json`, so let's save ourselves an extra PR, change it here and wait until that lands.

Reply:
Sure, I'll wait and move the `--json` flag.

Reply:
I've amended the code to use `edb cli query -Fjson` as per latest CLI implementation.